book/website/communication/binder/zero-to-binder.md

```diff
@@ -0,0 +1,525 @@
+
+Thank you to Anna Krystalli and Oliver Strickson for their help developing the R and Julia content, respectively.
+```
+
+```{attention}
+Binder can take a long time to load, but this doesn't necessarily mean that your Binder will fail to launch.
+You can always refresh the window if you see the "... is taking longer to load, hang tight!" message.
+```
+
+```{admonition} If you are using R...
+If you are following the R path, we have included some alternative steps using the [`holepunch` package](https://github.com/karthik/holepunch) which will build your environment using a [rocker base image](https://github.com/rocker-org/rocker) and should, ultimately, be faster to build and launch.
+```
+
+(z2b-reqs)=
+## Requirements
+
+You will need:
+
+- **Some code and some data.**
+  The code should take less than **10 minutes to run**, and the data should be less than **10 MB**.
+  This might mean that you just pick one script from a bigger project, or bring a subset of your data.
+  Note that it's really important that the code and data can be made **public** because we'll be using the public binder instance.
+- **A GitHub account.**
+  Please sign up for one if you don't already have one: <https://github.com/join>
+
+(z2b-step-1)=
+## 1. Creating a repo to Binderize
+
+🚦🚦🚦
+
+````{tabbed} Python
+1) Create a new repo on GitHub called "my-first-binder".
+   - Make sure the repository is **public**, _not private_!
+   - Don't forget to initialise with a README!
+2) Create a file called `hello.py` via the web interface with `print("Hello from Binder!")` on the first line and commit to the `main` branch
+````
+
+````{tabbed} Julia
+1) Create a new repo on GitHub called "my-first-binder".
+   - Make sure the repository is **public**, _not private_!
+   - Don't forget to initialise with a README!
+2) Create a file called `hello.jl` via the web interface with `println("Hello from Binder!")` on the first line and commit to the `main` branch
+3) Create a file called `Project.toml` (:rotating_light: the capitalisation is important!) with the following content and commit it to `main`.
+   This will install Julia into the Binder environment.
+
+   ```julia
+   [compat]
+   julia = "1.3"
+   ```
+````
+
+````{tabbed} R
+1) Create a new repo on GitHub called "my-first-binder".
+   - Make sure the repository is **public**, _not private_!
+   - Don't forget to initialise with a README!
+2) Create a file called `hello.R` via the web interface with `print("Hello from Binder!")` on the first line and commit to the `main` branch.
+3) Create a file called `runtime.txt` with `r-2022-01-01` on the first line.
+   This date represents the snapshot of [CRAN](https://cran.r-project.org/) hosted on the [RStudio Package Manager](https://packagemanager.rstudio.com) we will use.
+   Commit this file to the `main` branch.
+
+   ```{note}
+   In R you can use `holepunch::write_runtime()` to create a `runtime.txt` in the `.binder/` directory, configured with today's date.
```

Comment:
```suggestion
   In R you can use `holepunch::write_runtime()` to create a `runtime.txt` in the `.binder/` directory; it will be configured with today's date.
```